analytics/src/components/SearchInput/SearchInput.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
How shall we be testing these awesome components? Shall we be adding files like SearchInput.test.js under the same directory? We could explore the usage of Jest and Enzyme for unit testing within our awesome project - I have come to understand that they complement one another when it comes to testing. :)

Reply:
Testing React with Jest and Enzyme: https://medium.com/codeclan/testing-react-with-jest-and-enzyme-20505fec4675